src/openapi/I_Test_Driver_FdV.yaml

```diff
@@ -1055,7 +1090,7 @@ components:
             - validTo
           properties:
             hci_DisplayName:
-              $ref: '#/components/schemas/DisplayNameType'
+              $ref: '#/components/schemas/DeviceDisplayNameType'
```

Comment:
Why there is a need to change the referenced object here? This would be a breaking change. This endpoint is not about registering a device. Is it maybe a copy/paste issue?

Reply:
change should be on PutRegisteredDeviceRequestDTO. I have fixed it.